### PR TITLE
Upgrade ruby

### PR DESCRIPTION
**Pull request by boblail**

### CODE REVIEW

File: hiccup.gemspec [hunk 1]
```diff
@@ -10,19 +10,23 @@ Gem::Specification.new do |s|
...
+  s.add_dependency "activemodel", ">= 3.2.8"
   s.add_dependency "builder"
-  
+
   s.add_development_dependency "ri_cal"
-  s.add_development_dependency "rails", ">= 3.2.8", "< 4.1.0"
-  s.add_development_dependency "turn"
+  s.add_development_dependency "bundler", "~> 1.10"
+  s.add_development_dependency "rake", "~> 10.0"
+  s.add_development_dependency "minitest", "~> 5.0"
```
Comment (ben5516) [inline hiccup.gemspec]: Doesn't Ruby include this by default?

Comment (boblail) [inline hiccup.gemspec]: I think so, but it's just like a gem — and this says that if minitest 6.0 comes out, hiccup isn't _necessarily_ compatible with it.

Comment (boblail) [inline hiccup.gemspec]: — and clients of hiccup could always `gem install` a newer version of minitest than came with Ruby

Comment (boblail) [inline hiccup.gemspec]: — I'm not sure `hiccup` really cares what version of minitest. That's more something that `minitest-turn_reporter` cares about. I dropped the line.

**Pull request merged**